source/draw/gpu/opengl/backend/gles3/Fbo.ooc

```diff
@@ -65,7 +64,7 @@ Fbo: class {
 		if (status != GL_FRAMEBUFFER_COMPLETE) {
 			statusMessage := getErrorMessage(status)
 			errorMessage := "Framebuffer Object creation failed with status: " + statusMessage + " for texture of size " +
-			texture width toString() + " x " + texture height toString()
+			texture size width toString() + " x " + texture size height toString()
```

Comment:
Yes, the error throws an exception so unless we catch it (which we don't) it will crash the application.